src/scripts.js

```diff
@@ -94,6 +94,27 @@ clearCookbookSearchButton.addEventListener('click', function () {
   clearSearchBar(cookbookSearchBar)
 });
 
+recipeSection.addEventListener('click', (event) => {
+  if (event.target === recipeSection) {
+    return
+  }
+
```

Comment:
Looks good! We can refactor this later if we have time to have a helper function (event handler). Otherwise looks great!